common/vars.go

```diff
@@ -40,6 +40,7 @@ Options:
     -m, --method <METHOD>            Rotation method (sequent/random) (default: sequent)
     -r, --rotate <N>                 Rotate proxy IP after N request (default: 1)
         --max-retries <N>            Max. retries for failed HTTP requests (default: 0)
+        --max-redirects <N>          Max. redirects for HTTP requests (default: 10)
```

Comment:
```suggestion
        --max-redirs <N>          Max. redirects allowed (default: 10)
```

^ Following the curl-style.

```console
$ curl --help all | grep "max-redirs"
     --max-redirs <num>                            Maximum number of redirects allowed
```

Also update the README accordingly.

Reply:
done